### ISSUE TITLE
Pre-course material/pre-requisites

### ISSUE THREAD

**Issue by rebecacalvoquintero** [foundersandcoders/master-reference#391]
I would have found useful to have  included in the pre-requisites support material: flexbox, semantic css and HTML 5.

They would have been extremely helpful when building my website.

**Comment by jsms90** [comment]
Hey @rebecacalvoquintero!! Thanks so much for the issue :tada: 
The evening after day 1 of the course :dancer: 

Would you mind opening this in [`selection-committee`](https://github.com/foundersandcoders/selection-committee/issues) please? The content of the pre-reqs are decided by that Committee.

If they end up changing the pre-reqs, the pre-material will definitely need to be updated though :+1:

**Comment by rebecacalvoquintero** [comment]
thanks so much @jsms90 !!! I have created that issue on selection-committee repo. Sorry I have misunderstood where to issue it!

See you tomorrow!

**Comment by jsms90** [comment]
@rebecacalvoquintero Thanks very much :smile: No apology necessary! The two are very related anyway.

It's actually making me wonder whether it's problematic that we're dealing the two things in separate places :thinking:

**Comment by rebecacalvoquintero** [comment]
Yes, I agree. If possible, I could delete this if you think it makes sense

**Comment by jsms90** [comment]
@rebecacalvoquintero given that FAC [will be introducing a "prep course"](https://github.com/foundersandcoders/selection-committee/issues/53) which is a combination of precourse & pre-reqs, I reckon the pre-reqs should start being discussed in `master-reference` after all #424 

So this can probably stay open after all. The `selection-committee` can just gradually migrate their attention to this repo :+1:
